messages/api.go

```diff
@@ -78,8 +68,8 @@ type EndpointMetric struct {
 
 	// This field is true if this distribution is not cumulative.
 	IsNotCumulative bool `json:"isNotCumulative,omitempty"`
-	// The buckets for the distribution.
-	Buckets []Range `json:"buckets,omitempty"`
+	// The upper limits for the distribution buckets.
+	UpperLimits []float64 `json:"upperLimits,omitempty"`
 }
```

Comment:
In my distributions, the first bucket never has a lower limit. My buckets are always something like:
<5, 5-10, 10-20, 20-50, >50.

So -Inf always goes in the first bucket.